src/elements/types/ObjectExpression.js

```diff
@@ -17,8 +17,8 @@ export default class ObjectExpression extends Expression {
                 children.skipNonCode();
                 children.assertToken('Punctuator', '}');
             } else {
-                if (children.isNode('SpreadProperty')) {
-                    properties.push(children.passNode('SpreadProperty'));
+                if (children.isNode('SpreadElement')) {
+                    properties.push(children.passNode('SpreadElement'));
```

Comment:
`SpreadProperty` and `RestProperty` have been removed.

https://babeljs.io/docs/en/v7-migration-api#restproperty-spreadproperty